core-v1/src/main/scala/dsl/WrapSelected.scala

```diff
@@ -30,7 +30,6 @@ trait WrapSelected[W[+_], OP[_]] {
     path: DataPath,
     element: O,
   )(implicit
-    opA: OP[I],
```

Comment:
This never really made sense... `OP` stands for "output parameter" and this is being applied to the *input*.